README.md

```diff
@@ -524,6 +538,22 @@ Output:
 }
 ```
 
+Usage (Proc Option):
+```ruby
+class UserBlueprint < Blueprinter::Base
+  identifier :name
+  field :birthday, datetime_format: ->(datetime) { datetime.nil? ? datetime : datetime.strftime("%s").to_i }
+end
+```
+
+Output:
+```json
+{
+  "name": "John Doe",
+  "birthday": 762739200
+}
+```
+
```

Comment:
Might be nice to add a clarifying line such as what is on [Line 506](https://github.com/procore/blueprinter/pull/145/files#diff-04c6e90faac2675aa89e2176d2eec7d8R506)